example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -33,9 +54,72 @@ protected void onCreate(Bundle savedInstanceState) {
         settings.setRetryOrder(RetryOrder.Queue);
 
         BacktraceDatabase database = new BacktraceDatabase(context, settings);
-        BacktraceClient backtraceClient = new BacktraceClient(context, credentials, database);
+        backtraceClient = new BacktraceClient(context, credentials, database);
 
         BacktraceExceptionHandler.enable(backtraceClient);
         backtraceClient.send("test");
+
+        // Enable handling of native crashes
+        database.setupNativeIntegration(backtraceClient, credentials);
+
+        // Enable ANR detection
+        backtraceClient.enableAnr(anrTimeout);
+    }
+
+    public native void cppCrash();
+
+    private ArrayList<String> equippedItems;
+
+    public ArrayList<String> getWarriorArmor()
+    {
+        return new ArrayList<String>(Arrays.asList("Tough Boots", "Strong Sword", "Sturdy Shield", "Magic Wand"));
```

Comment:
It won't be better if you will use:
` List.of("Tough Boots", "Strong Sword", "Sturdy Shield", "Magic Wand");`?

Also - we don't need to pack it once again into `ArrayList` - `Arrays.asList("Tough Boots", "Strong Sword", "Sturdy Shield", "Magic Wand")` will generate List of strings.

Reply:
@konraddysput we are currently using Java 8 so we don't have `List.of` method available.

Also if I don't pack again into ArrayList, I have to cast it like this:
`return (ArrayList<String>) Arrays.asList("Tough Boots", "Strong Sword", "Sturdy Shield", "Magic Wand");`

I think because we are using older Java version we are stuck with this way for now.